### PR TITLE
Add Compensation and Job Application serverless workflow samples

### PR DESCRIPTION
**Pull request by caponetto**
Adding two more serverless workflow samples.
I also removed some undesirable dashed borders (top and left) from the greeting SVG.

### CODE REVIEW

**Comment by caponetto** [comment]
@adrielparedes can you please review this one?
Thanks :)

File: samples/compensation-json/compensation.sw.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+  "id": "compensation",
+  "version": "1.0",
+  "specVersion": "0.8",
+  "name": "Workflow Error example",
```
Comment (adrielparedes) [inline samples/compensation-json/compensation.sw.json:5]: The capitalization of those words is ok?

File: samples/compensation-json/compensation.sw.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        "compensated": true
+      },
+      "transition": "compensating_more"
+    },
+    {
+      "name": "compensating_more",
+      "usedForCompensation": true,
+      "type": "inject",
+      "data": {
+        "compensating_more": "Real Betis Balompie"
```
Comment (adrielparedes) [inline samples/compensation-json/compensation.sw.json:55]: hahaha why?

File: samples/compensation-json/definition.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "type": "sample",
+  "category": "serverless-workflow",
+  "status": "ok",
+  "title": "Workflow Error example",
```
Comment (adrielparedes) [inline samples/compensation-json/definition.json:5]: Same here?

File: samples/job-application-event-json/definition.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "type": "sample",
+  "category": "serverless-workflow",
+  "status": "ok",
+  "title": "Applicant Workflow",
+  "description": "Job Application Events Example",
+  "cover": "./applicant-workflow.sw.json.svg",
+  "tags": ["serverless workflow", "json", "events"],
```
Comment (adrielparedes) [inline samples/job-application-event-json/definition.json]: add also "swf" tag please

**Review by adrielparedes — CHANGES_REQUESTED** [review]

File: samples/compensation-json/compensation.sw.json (hunk 2 above)
Comment (caponetto) [reply samples/compensation-json/compensation.sw.json:55]: These samples are from `kogito-examples` repo. The only things I changed were some minor stuff to fix validation warnings.

File: samples/job-application-event-json/definition.json (hunk 4 above)
Comment (caponetto) [reply samples/job-application-event-json/definition.json]: Will do it in the next commit!

Comment (caponetto) [reply samples/job-application-event-json/definition.json]: Done in [0b1734d](https://github.com/kiegroup/kie-samples/pull/8/commits/0b1734d4c1cdb6306e7e85b2b99eb76d9a060360)

File: samples/compensation-json/compensation.sw.json (hunk 1 above)
Comment (caponetto) [reply samples/compensation-json/compensation.sw.json:5]: I kept everything from kogito-examples repo. Should we change it to something else?

**Review by adrielparedes — APPROVED** [review]

**Comment by adrielparedes** [comment]
@caponetto let's wait for the green build, everything is fine!

**Pull request merged**